eox_tenant/views.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""The generic views for the eox-tenant plugin project"""
+
+from __future__ import unicode_literals
+
+import json
+from os.path import dirname, realpath
+from subprocess import check_output, CalledProcessError
+
+from django.http import HttpResponse
+from django.utils import six
+
+
+import eox_tenant
+
+
+def info_view(request):  # pylint: disable=unused-argument
+    """
+    Basic view to show the working version and the exact git commit of the
+    installed app.
+    """
+    try:
+        working_dir = dirname(realpath(__file__))
+        git_data = six.text_type(check_output(
```

Comment:
imo the line doesn't need this break, it's not long enough

Reply:
Without the line break it exceeds 80 lines

Reply:
we use 120 as the max number of characters https://github.com/eduNEXT/eox-tenant/blob/master/setup.cfg#L17